src/marshal.rs

```diff
@@ -21,15 +21,14 @@ pub const VERSION: i32 = 4;
 /// # Examples
```

Comment:
This file seems accidentally changed mode from `644` to `755`

Reply:
Oops, thank you for pointing out that!